concrete_datastore/concrete/apps.py

```diff
@@ -52,12 +57,52 @@ def purge_users_with_wrong_usernames(user_model):
                 archive_legacy(user=user, email_list=email_list)
 
 
+def alter_migration_content(file):
```

Comment:
Why do we need to alter existing migrations ? I think we can avoid this

Reply:
in the existing migrations, we may find an import of `pendulum.pendulum` (pendulum < v2.0). With the v2 of pendulum, django commands crash with `ModuleNotFoundError: No module named 'pendulum.pendulum'`

Reply:
As this code modifies python source files (migration files), a complete restart is required so we cannot let `alter_migration_content` in the module `concrete_datastore/concrete/apps.py`.
Furthermore, it will require to clear the pyc / __pycache__ and that can be an issue.

Other solutions can be discussed:
- a management command 
- a hook in the migration module to patch migrations at runtime
- no automatic rewrite of migrations but the script can propose a manual action to the developer with the required patch (explicit)

Other (bad) edge cases can arise from modifiying source files from within `concrete`.

Reply:
this will be handeled in another script to fix migrations